app/actions/remote/post.test.ts

```diff
@@ -419,8 +603,30 @@ describe('get posts', () => {
         expect(result.posts?.length).toBe(2);
     });
 
+    it('fetchPosts - no CRT', async () => {
+        await operator.handleSystem({systems: [{id: SYSTEM_IDENTIFIERS.CURRENT_USER_ID, value: user1.id}], prepareRecordsOnly: false});
+        mockGetIsCRTEnabled.mockImplementationOnce(() => false);
+
+        const result = await fetchPosts(serverUrl, channelId);
+        expect(result).toBeDefined();
+        expect(result.error).toBeUndefined();
+        expect(result.posts).toBeTruthy();
+        expect(result.posts?.length).toBe(2);
+    });
+
+    it('fetchPosts - no authors needed', async () => {
+        await operator.handleSystem({systems: [{id: SYSTEM_IDENTIFIERS.CURRENT_USER_ID, value: user1.id}], prepareRecordsOnly: false});
+        mockClient.getProfilesByIds.mockImplementationOnce(jest.fn(() => []));
+
+        const result = await fetchPosts(serverUrl, channelId);
+        expect(result).toBeDefined();
+        expect(result.error).toBeUndefined();
+        expect(result.posts).toBeTruthy();
+        expect(result.posts?.length).toBe(2);
+    });
+
     it('fetchPostsBefore - handle database not found', async () => {
-        const result = await fetchPostsBefore('foo', '', '') as {error: unknown};
+        const result = await fetchPostsBefore('foo', '', '', 50, true) as {error: unknown};
```

Comment:
just a thought, would it be good to use the constans declared General.POST_CHUNK_SIZE, rather than hard coding?

Reply:
I think it would if we were doing an `expect` here but since it shouldn't impact the result of the test I think it's ok to leave as is